shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-core/src/main/java/org/apache/shardingsphere/sharding/rule/ShardingRule.java

```diff
@@ -279,13 +279,13 @@ private String getAlgorithmExpression(final TableRule tableRule, final boolean d
     }
     
     private String getShardingColumn(final ShardingStrategyConfiguration shardingStrategyConfig) {
+        String shardingColumn = this.defaultShardingColumn;
```

Comment:
Here this is not necessary.

Reply:
It can't be like this:

```
private String getShardingColumn(final ShardingStrategyConfiguration shardingStrategyConfig) {
        String shardingColumn = this.defaultShardingColumn;
        if (shardingStrategyConfig instanceof ComplexShardingStrategyConfiguration) {
            shardingColumn = ((ComplexShardingStrategyConfiguration) shardingStrategyConfig).getShardingColumns();
        }
        if (shardingStrategyConfig instanceof StandardShardingStrategyConfiguration) {
            shardingColumn = ((StandardShardingStrategyConfiguration) shardingStrategyConfig).getShardingColumn();
        }
        return StringUtils.defaultString(shardingColumn, "");
}
```

Reply:
![image](https://user-images.githubusercontent.com/12792261/164181656-65c9b72c-15ee-4c2e-bd28-4f05ccd889d7.png)
`this` is unnecessary.